src/components/DisplayComponent.js

```diff
@@ -254,13 +299,47 @@ class DisplayComponent extends Component {
         <Normalization />
         <ToolkitProvider keyField="id" data={data} columns={columns} exportCSV>
           {props => (
-            <div>
+            <div className="">
```

Comment:
Empty `className`

Reply:
here, i want to add something to move the div to the line above. I'm unsure of what, though

Reply:
I think this is taken care of in #136

Reply:
Then I'll remove it for now.